### ISSUE TITLE
filtered importer bug

### ISSUE THREAD

**Issue by NarigudoBR** [MrTJP/ProjectRed#852]
I was testing a way to remove broken pickaxes from arcane bore,to my frame system so when I activate the filtered importer it imported the filtered items(ok),but it imported the not filtered items too,and without no reason the dessapeared too(the not filtered).

**Comment by the-creamster** [comment]
I've also found this to be true when using them to insert/extract batteries(both full and empty) to/from an enderchest and battery box. I figured out the isidedness is bottom for empty, top for fill and it'll work with enderio conduits but not for the pressure tubes.

Also it would be great if a redstone pressure tube could be added as i'm in the process of building a replication of a redpower2 6-axis motion unit which works perfectly but the current hitches are:
1. filtered importer not working preventing me from remotely powering it
2. AE can be used as the mining head but not being able to put one of its cables through the frame block made me use the block breaker instead but won't be able to activate all of them without having the redstone on bottom of the mining head itself which means a loss in mining efficiency which currently will need a 3 step movement motion just to mine a single layer
3. When using atleast 2 multiparts on a frame plus an inserted part a third can't be placed depending upon the multiparts being used (my case is a framed low load power line inside the frame with a marble brick cover on top, basalt brick panel on the left and want to put a hollow basalt brick panel on the bottom but it doesn't let me), bug or intended to have a multipart cap??

When i was building this 6-axis motion unit which is 6 long*4 wide*5 high i discovered various hidden gems for me atleast which simpified the whole build process(tubes, framed wires & power cables being insertable into the frame block itself) it amazed me and worked really nice so i take my hat off to you for that.

Another minor annoyance is that the frame blocks have the smallest place/mine hitbox. Is that intended so various things can be inserted into them for ease of building and compaction or a bug??

Thanks, The_Creamster.
